src/tokens.js

```diff
@@ -1132,6 +1132,103 @@ const tokens = {
     return uid === HATHOR_TOKEN_CONFIG.uid;
   },
 
+  /**
+   * Checks if this transaction is the creation of an NFT following the NFT Standard Creation.
+   * @see https://github.com/HathorNetwork/rfcs/blob/master/text/0032-nft-standard.md#transaction-standard
+   * @param {Object} tx Transaction with decoded inputs and outputs
+   */
+  isNFTToken(tx) {
+    // Basic input validation
+    if (!tx) {
+      return false;
+    }
+
+    // If this is not a token creation tx, it cannot be an NFT
+    if (tx.version !== CREATE_TOKEN_TX_VERSION) {
+      return false;
+    }
+
+    // NFT creation must have at least a DataScript output (the first one) and a Token P2PKH output
+    if (tx.outputs.length < 2) {
+      return false;
+    }
+
```

Comment:
You should add here a `DataScript` validation for the first output. Checking the `value` and `token_data` only is not enough.

We have a method `parseScriptData` in `utils/scripts.ts`, which receives the `script` in `Buffer` and throws an error in case it's not a data script.

We should add this method here as well and keep the value/token_data validations.